### DISCUSSION TITLE
Secrets plugin: 'Activate selected user keys' error

### DISCUSSION THREAD

**Discussion by vapecastle** [netbox-community/netbox#14457] [Help Wanted!]
Hi All,

Not sure if I have set things up right here or not, but when I activate a user key for any user I get the following error. Have tried adding a new user to see if the issue was with the user itself, but I get the same result.

Any ideas? Do I need to provide better debugging info?

<class 'ValueError'>

Field 'id' expected a number but got ''.

Python version: 3.10.12
NetBox version: 3.6.5
Plugins: 
  netbox_secrets: 1.9.0

**Comment by abhi1693** [comment]
https://github.com/netbox-community/netbox/issues/14392
